packages/design-system/infinite-scroll-list/infinite-scroll-list.web.tsx

```diff
@@ -258,6 +261,10 @@ function InfiniteScrollListImpl<Item>(
               </div>
               {renderedItems.map((virtualItem) => {
                 const index = virtualItem.index;
+                const chuckItem = data?.slice(
+                  index * numColumns,
+                  index * numColumns + numColumns
+                );
```

Comment:
ChunkItem again? We had perf issues with this in the past and we switched like to use grid, right?

Reply:
No, this is only a placeholder because we currently have a spacing issue with numColumns on the web.